### ISSUE TITLE
It can be trained but not predictable

### ISSUE THREAD

**Issue by goodsave** [milesial/Pytorch-UNet#201]
I cloned the project and changed n_ Channels = 1 configuration.
No other changes were made. The dice value was 0.8560 during training,
The label values of my training set are 0 and 1 respectively,
After prediction, it is found that all the label values are 0. What is the reason?

**Comment by goodsave** [comment]
![406c9310895fc396771295a0805a1a7](https://user-images.githubusercontent.com/19647156/88249489-3526ed80-ccd7-11ea-871b-188fbdd5aeb3.png)
![8ce10721d203a28af1e5a2ad64b73fa](https://user-images.githubusercontent.com/19647156/88249492-36581a80-ccd7-11ea-869f-d23d1c768ace.png)
![3111f521a6534521560eb1e187c2d7e](https://user-images.githubusercontent.com/19647156/88249526-4bcd4480-ccd7-11ea-995c-d518117f941e.png)

**Comment by milesial** [comment]
Have you tried predicting without any threshold and saving the greyscale mask?

**Comment by goodsave** [comment]
Thank you for your reply。
I haven't changed the threshold for the time being.
What parameters do I need to change for debugging?

**Comment by milesial** [comment]
The threshold, mainly, to see if the output probs make sense

**Comment by lightersss** [comment]
我也遇到了相同的情况，在output的文件是黑白有mask的，但是用PIL展示就是全紫色的。

**Comment by milesial** [comment]
Closing, as this issue is now outdated after the recent code refactor.
